language/tools/move-mv-llvm-compiler/tests/test_common.rs

```diff
@@ -11,8 +11,22 @@ use std::{
     fs,
     path::{Path, PathBuf},
     process::Command,
+    sync::atomic::{AtomicBool, Ordering},
 };
 
+/// Atomic used to prevent re-initialization of logging.
+static LOGGER_CONFIGURED: AtomicBool = AtomicBool::new(false);
+
+pub fn setup_logging_for_test() {
+    if LOGGER_CONFIGURED
+        .compare_exchange(false, true, Ordering::Relaxed, Ordering::Relaxed)
+        .is_err()
+    {
+        return;
+    }
+    env_logger::init();
+}
+
```

Comment:
I realize you are moving existing code, butI I think `std::sync::Once` (with `call_once`)  is a more natural/Rust way to do this. There is an example already in test_common.rs. Suggestion only.

Reply:
Good point. Indeed, it's better to use the idiom. I updated the PR.